spec/requests/acuant_sdk_spec.rb

```diff
@@ -15,7 +15,7 @@
 
   it 'has between 2 and 3 versions' do
     sdk_versions_count = acuant_entries.count { |entry| entry =~ /[0-9]+(\.[0-9]+)+/ }
-    expect(sdk_versions_count).to be_between(2, 3)
+    expect(sdk_versions_count).to be_between(2, 4)
```

Comment:
IIRC this test was intentionally added to keep ourselves accountable to deleting old unused versions. Are we able to remove an old version with this update?

Reply:
I increased it since two of the versions are patches of 11.9.3. I wasn't sure if we should delete 11.9.2 since it was the last main version.

Reply:
Do we still use that version in production? If not, I think it can be removed.